src/MetaDataAPI/Models/Response/Converters/Erc721AttributeConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using MetaDataAPI.Models.Types;
+
+namespace MetaDataAPI.Models.Response.Converters;
+
+public class Erc721AttributeConverter : JsonConverter<Erc721Attribute>
+{
+    public override Erc721Attribute ReadJson(JsonReader reader, Type objectType, Erc721Attribute? existingValue, bool hasExistingValue, JsonSerializer serializer)
+    {
+        var jsonObject = JObject.Load(reader);
+        var traitType = jsonObject["trait_type"]!.Value<string>()!;
+        var value = jsonObject["value"]!.ToObject<object>()!;
+        var displayTypeString = jsonObject["display_type"]?.Value<string>();
+
+        var displayType = DisplayType.String;
+        if (displayTypeString != null)
+        {
+            Enum.TryParse(displayTypeString, true, out displayType);
+        }
+
+        return new Erc721Attribute(traitType, value, displayType);
+    }
+
+    public override void WriteJson(JsonWriter writer, Erc721Attribute? value, JsonSerializer serializer)
+    {
+        var obj = new JObject
+        {
+            ["trait_type"] = value!.TraitType,
+            ["value"] = JToken.FromObject(value.Value)
+        };
+
+        if (value.DisplayType != DisplayType.String)
+        {
+            obj["display_type"] = value.DisplayType.ToString().ToLowerInvariant();
+        }
+
+        obj.WriteTo(writer);
+    }
+}
```

Comment:
can we use the same class? why the default reader can't read it?
we can save in any format, we can save in format that will be more "direct"

Reply:
1. Reader cannot correctly read `string` `display_type`.
![image](https://github.com/The-Poolz/LockDeal-NFT-API/assets/66203238/b4677634-f3d2-4c29-9f3d-f7494ca56eae)
2. I need to parse `display_type` of attribute, to work with it. Better parse it once, instead of parse it each time.

Reply:
File deleted in #123